ui/components/MesherySMIResults.js

```diff
@@ -227,7 +227,7 @@ class MesherySMIResults extends Component {
                     </TableHead>
                     <TableBody>
                       {data?.map((row) => (
-                        <TableRow >
+                        <TableRow key={row}>
```

Comment:
If the results are dynamic which in this case it would be , here key should be `key={row.uniqueID}`

Reply:
Ok i'm pushing the change